ibis/backends/clickhouse/tests/test_client.py

```diff
@@ -359,3 +365,10 @@ def test_password_with_bracket():
         cc.driver.exceptions.DatabaseError, match="password is incorrect"
     ):
         ibis.clickhouse.connect(host=host, user=user, port=port, password=quoted_pass)
+
+
+def test_invalid_port(con):
+    port = 9999
```

Comment:
Can we add a successful connection using the correct port at the beginning of this test to show that the port is being correctly parsed?

Reply:
Nice idea. I will add a successful test case. Thanks.